.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
         {
```

Comment:
Probably, this file is not related to the PR

Reply:
Sure, will move the config improvement to a separate PR